README.md

```diff
@@ -54,9 +54,21 @@ we are trying to [dogfood](#the-dogfooding-cluster) more and more.
 
 Secrets which have been applied to the prow cluster but are not committed here are:
 
-- `GitHub` secrets:
+- `GitHub` personal access tokens:
  - `bot-token-github` in the default namespace
  - `bot-token-github` in the github-admin namespace
+ - `ouath-token` in the default namespace
```

Comment:
nit: ouath

also this is a dupe of `oauth-token` on line 62 now i think?

Reply:
😅  whoops, fixed!